### ISSUE TITLE
model.eval() starts training session

### ISSUE THREAD

**Issue by aiexperts-andreas** [lyuwenyu/RT-DETR#80]
Hi, my code looks like this:

_from ultralytics import RTDETR

model = RTDETR(model = str('eide_pelletpoc_RT-DETR/runs/detect/eide_train_3k_fix/weights/best.pt'))

model.eval()_

I am using a model that I have trained.
Why does model.eval() start a training session on default parameters?
Pastebin of log when I start running model.eval():

_New https://pypi.org/project/ultralytics/8.0.196 available  Update with 'pip install -U ultralytics'
Ultralytics YOLOv8.0.193  Python-3.11.4 torch-2.0.1+cpu CPU (AMD Ryzen 7 5825U with Radeon Graphics)
engine\trainer: task=detect, mode=train, model=eide_pelletpoc_RT-DETR/runs/detect/eide_train_3k_fix/weights/best.pt, data=coco8.yaml, epochs=100, patience=50, batch=16, imgsz=640, save=True, save_period=-1, cache=False, device=None, workers=8, project=None, name=None, exist_ok=False, pretrained=True, optimizer=auto, verbose=True, seed=0, deterministic=True, single_cls=False, rect=False, cos_lr=False, close_mosaic=10, resume=False, amp=True, fraction=1.0, profile=False, freeze=None, overlap_mask=True, mask_ratio=4, dropout=0.0, val=True, split=val, save_json=False, save_hybrid=False, conf=None, iou=0.7, max_det=300, half=False, dnn=False, plots=True, source=None, show=False, save_txt=False, save_conf=False, save_crop=False, show_labels=True, show_conf=True, vid_stride=1, stream_buffer=False, line_width=None, visualize=False, augment=False, agnostic_nms=False, classes=None, retina_masks=False, boxes=True, format=torchscript, keras=False, optimize=False, int8=False, dynamic=False, simplify=False, opset=None, workspace=4, nms=False, lr0=0.01, lrf=0.01, momentum=0.937, weight_decay=0.0005, warmup_epochs=3.0, warmup_momentum=0.8, warmup_bias_lr=0.1, box=7.5, cls=0.5, dfl=1.5, pose=12.0, kobj=1.0, label_smoothing=0.0, nbs=64, hsv_h=0.015, hsv_s=0.7, hsv_v=0.4, degrees=0.0, translate=0.1, scale=0.5, shear=0.0, perspective=0.0, flipud=0.0, fliplr=0.5, mosaic=1.0, mixup=0.0, copy_paste=0.0, cfg=None, tracker=botsort.yaml, save_dir=runs\detect\train14
Overriding model.yaml nc=1 with nc=80_
